src/shiver/models/histogram.py

```diff
@@ -130,9 +131,81 @@ def rename(self, old_name, new_name):
         RenameWorkspace(old_name, new_name)
 
     def save(self, ws_name, filename):
-        """Save the workspace"""
+        """Save the workspace to Nexus file."""
         SaveMD(ws_name, filename)
 
+    def save_to_ascii(
+        self,
+        ws_name: str,
+        filename: str,
+        ignore_integrated: bool = True,
```

Comment:
Please change the default to False

Reply:
The default is switched to False in the latest commit.